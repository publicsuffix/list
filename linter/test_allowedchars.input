// test:
// - label contains illegal character
// - c. is valid UTF-8
// - d. has invalid UTF-8 code for the TLD
//
// best viewed with 'LC_ALL=C vi <filename>'

// ===BEGIN ICANN DOMAINS===

a.exam#ple.com
b.exam ple.com
c.æµ‹è¯•
d.æµ‹è¯

// ===END ICANN DOMAINS===
