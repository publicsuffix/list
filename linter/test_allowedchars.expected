10: error: Illegal character: 'a.exam#ple.com'
11: error: Illegal character: 'b.exam ple.com'
13: error: Invalid UTF-8 character
15: warning: No PRIVATE section found
